### ISSUE TITLE
Support Interface Staging for 6 GHz Frequencies for Passive Mode

### ISSUE THREAD

**Issue by joshschmelzle** [WLAN-Pi/profiler#85]
While we haven't figured out injection on 6 GHz yet, we can passively listen for association requests from clients to APs in the 6 GHz mid-band spectrum.

* Note this requires 1) interface staging and scanning, 2) correct hardware/firmware/software and 3) `--noAP` and `--noprep` options to be passed in, **all prior** to profiler being launched.

With that said, 6 GHz raises some questions and a need to do some investigation on behavior and testing of the profiler in 6 GHz.

* Is it time to switch the channel configuration from channel # to frequency? Instead of channel 1 in 2.4 GHz, using 2412. Instead of channel 36 in 5 GHz, using 5180. For channel 5 in 6 GHz, 5975.

**Comment by joshschmelzle** [comment]
When we get an association frame, print better debugs. For example in 5 GHz, it's not clear where we get the ch and freq from.

```
2021-11-08 14:17:44,201 [DEBUG] fakeap.py: adding assoc req from 6e:1d:8a:28:32:51 to queue
2021-11-08 14:17:44,232 [DEBUG] fakeap.py: adding assoc req from 6e:1d:8a:28:32:51 to queue
2021-11-08 14:17:44,247 [DEBUG] profiler.py: freq is 5180
2021-11-08 14:17:44,247 [DEBUG] profiler.py: ch is 36
```

When we generate a "profile", the listed channel and frequency are from the association frame.

**Comment by joshschmelzle** [comment]
> When we get an association frame, print better debugs. For example in 5 GHz, it's not clear where we get the ch and freq from.
> 
> ```
> 2021-11-08 14:17:44,201 [DEBUG] fakeap.py: adding assoc req from 6e:1d:8a:28:32:51 to queue
> 2021-11-08 14:17:44,232 [DEBUG] fakeap.py: adding assoc req from 6e:1d:8a:28:32:51 to queue
> 2021-11-08 14:17:44,247 [DEBUG] profiler.py: freq is 5180
> 2021-11-08 14:17:44,247 [DEBUG] profiler.py: ch is 36
> ```
> 
> When we generate a "profile", the listed channel and frequency are from the association frame.

This has been addressed in #87

**Comment by joshschmelzle** [comment]
Tightening the scope of this issue to only deal with the input and staging of 6 GHz frequencies on iwlwifi ax210 cards.
